docs/versioned_docs/version-zenith/user/quickstart/292861-export.mdx

```diff
@@ -0,0 +1,40 @@
+---
+slug: /user/quickstart/292861/export
+hide_table_of_contents: true
+title: "Export function outputs"
+---
+
+# Quickstart
+
+## Export function outputs
+
+The `export` API allows you to retrieve file, directory and container outputs from functions and write them to the host filesystem.
+
+To see it in action, use `export` with the `build` function used previously, to export the built application to an `out` directory on the host:
+
+```shell
+dagger call build export --path ./out
+```
+
+Here's an example of the output:
+
+```shell
+✔ dagger call build export [31.79s]
+┃ Asset exported to "./out"
+$ ls out/
+asset-manifest.json  index.html   logo512.png    robots.txt
+favicon.ico          logo192.png  manifest.json  static
+```
+
+The `export` API also works with functions that return `Container` types, in which case the `Container` is exported as a TAR file to the host file system. Here's an example:
+
+```shell
+dagger call app-container export --path ./container.tar
```

Comment:
Fails

Reply:
Yes, I did so already https://github.com/dagger/dagger/pull/6548#issuecomment-1921052528 not

Reply:
Fixed now